lib/livebook_web/views/error_view.ex

```diff
@@ -2,7 +2,7 @@ defmodule LivebookWeb.ErrorView do
   use LivebookWeb, :view
 
   def render("401.html", %{reason: %LivebookWeb.InvalidTokenError{}} = assigns) do
-    render("401_token.html", assigns)
+    render("401_secret.html", Map.put(assigns, :type, :token))
```

Comment:
We probably don't need this anymore. The `authentication_mode` is either token or password, so I would rename it to 401.html and remove this clause. Then in the template you can check the authentication mode and act accordingly.